### PR TITLE
Support Scala 3.1 in sandwich scenario (#66)

### PR DESCRIPTION
**Pull request by DamianReeves**
Fix for #66

### CODE REVIEW

File: src/main/scala/sbt/VirtualAxis.scala [hunk 1]
```diff
@@ -65,7 +65,7 @@ object VirtualAxis {
 
   private[sbt] def isScala2Scala3Sandwich(sbv1: String, sbv2: String): Boolean = {
     def compare(a: String, b: String): Boolean =
-      a == "2.13" && (b.startsWith("0.") || b.startsWith("3.0"))
+      a == "2.13" && (b.startsWith("0.") || b.startsWith("3.0") || b.startsWith("3.1"))
```
Comment (eed3si9n) [inline src/main/scala/sbt/VirtualAxis.scala]: Maybe we should just make it `3.` for now?

**Review by eed3si9n — APPROVED** [review]
Thanks!

**Comment by DamianReeves** [comment]
Would be great to get this merged since 3.1 doesn't work now

**Comment by DamianReeves** [comment]
Thanks for merging. Does someone have to manually kick off a build to get a release for this?

**Pull request merged**